### PR TITLE
Second-Milestone: Update animals table

### PR DESCRIPTION
**Pull request by saluumaa**
In this Milestone I:- 

1. Add a species column 
2. Alter animals data and add new data
3. Add transactions to alter the animals' table
4. Take screenshots for transaction actions and queries
Animal#
Average-weight
dateofbirth-species
DeleteAll
escape#
neutered#
species_weight
update Weight
UpdateSpecies
makedecimal

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -9,3 +9,36 @@ SELECT * FROM animals WHERE neutered = true;
 SELECT * FROM animals WHERE name NOT IN ('Gabumon');
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
```
Comment (zieeco) [inline queries.sql:11]: ### schema.sql

- Hey, kindly add a new column called `species` with a string data type to meet this project requirement.

`ALTER TABLE animals ADD species varchar(255);`

File: data.sql [hunk 2]
```diff
@@ -14,4 +14,25 @@ INSERT INTO animals (name, data_of_birth, escape_attempts, neutered, weight_kg)
 VALUES ('Pikachu', '01-07-2021', 1, false, 15.04);
 
 INSERT INTO animals (name, data_of_birth, escape_attempts, neutered, weight_kg) 
-VALUES ('Devimon', '05-12-2017', 5, true, 11);
\ No newline at end of file
+VALUES ('Devimon', '05-12-2017', 5, true, 11);
+
+INSERT INTO animals (name, data_of_birth, escape_attempts, neutered, weight_kg) 
+VALUES ('Charmander', '2020-02-08', 0, false, -11);
```
Comment (zieeco) [inline data.sql]: - Hey, the data type for the weight is a floating number(decimal). let's ensure we've passed the correct data type to this column `'-11.0'`

File: data.sql [hunk 3]
```diff
@@ -14,4 +14,25 @@ INSERT INTO animals (name, data_of_birth, escape_attempts, neutered, weight_kg)
...
+
+INSERT INTO animals (name, data_of_birth, escape_attempts, neutered, weight_kg) 
+VALUES ('Plantmon', '2021-09-15', 2, true, -5.7);
+
+INSERT INTO animals (name, data_of_birth, escape_attempts, neutered, weight_kg) 
+VALUES ('Squirtle','1993-04-02', 3, false, -12.3);
+
+INSERT INTO animals (name, data_of_birth, escape_attempts, neutered, weight_kg) 
+VALUES ('Angemon', '2005-06-12', 1, true, -45);
```
Comment (zieeco) [inline data.sql]: - Hey, the data type for the weight is a floating number(decimal). let's ensure we've passed the correct data type to this column `'-45.0'`

File: data.sql [hunk 4]
```diff
@@ -14,4 +14,25 @@ INSERT INTO animals (name, data_of_birth, escape_attempts, neutered, weight_kg)
...
+
+INSERT INTO animals (name, data_of_birth, escape_attempts, neutered, weight_kg) 
+VALUES('Boarmon', '2005-06-07', 7, true, 20.4);
+
+INSERT INTO animals (name, data_of_birth, escape_attempts, neutered, weight_kg) 
+VALUES ('Blossom', '1998-10-13', 3, true, 17);
```
Comment (zieeco) [inline data.sql]: - Hey, the data type for the weight is a floating number(decimal). let's ensure we've passed the correct data type to this column `'17.0'`

File: data.sql [hunk 5]
```diff
@@ -14,4 +14,25 @@ INSERT INTO animals (name, data_of_birth, escape_attempts, neutered, weight_kg)
...
+
+INSERT INTO animals (name, data_of_birth, escape_attempts, neutered, weight_kg) 
+VALUES ('Ditto', '2022-05-14', 4, true, 22);
```
Comment (zieeco) [inline data.sql]: - Hey, the data type for the weight is a floating number(decimal). let's ensure we've passed the correct data type to this column `'22.0'`

File: data.sql [hunk 6]
```diff
@@ -14,4 +14,25 @@ INSERT INTO animals (name, data_of_birth, escape_attempts, neutered, weight_kg)
...
-VALUES ('Devimon', '05-12-2017', 5, true, 11);
\ No newline at end of file
+VALUES ('Devimon', '05-12-2017', 5, true, 11);
+
+INSERT INTO animals (name, data_of_birth, escape_attempts, neutered, weight_kg) 
+VALUES ('Charmander', '2020-02-08', 0, false, -11);
+
+INSERT INTO animals (name, data_of_birth, escape_attempts, neutered, weight_kg) 
+VALUES ('Plantmon', '2021-09-15', 2, true, -5.7);
+
```
Comment (zieeco) [inline data.sql]: - Hey, let's try to pay attention to details as the date is not correct. kindly update your date to `'2021/11/15',`

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @saluumaa,





Great job so far! 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice schema file. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._




## Optional suggestions
- N/A


**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by BregornOriginal — APPROVED** [review]
Hi @saluumaa 

Your project is complete!  There is nothing else to say other than... it's time to merge it 

Congratulations! 

giphy

## Highlights

✔️ Linters are passing
✔️ Good commit history
✔️ Good work implementing changes requested by the previous reviewer 💯 

## Optional suggestions
Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.
Cheers and Happy coding!:aplauso::aplauso::aplauso:
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

Please, remember to tag me in your question so I can receive the notification.
------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)[.](https://airtable.com/shrQAqnBwek5a0O0s)._)

**Pull request merged**